hypothesis-python/tests/py3/test_unicode_identifiers.py

```diff
@@ -40,3 +40,10 @@ def bar():
         return 2
 
     assert bar() == 2
+
+
+is_approx_π = lambda x: x == 3.1415  # noqa: E731
```

Comment:
Running locally, I can see that flake8 is upset about this line, because it thinks assigning a lambda expression is a bug. (And it sort of has a point, but it's the simplest way to hit this bug.)

I have no idea how you'd work that out from the Travis output: https://travis-ci.org/HypothesisWorks/hypothesis/jobs/469705820

Reply:
Wow.  [The report appears *before* the command...](https://travis-ci.org/HypothesisWorks/hypothesis/jobs/469705820#L784-L787) I guess buffering issues?